_episodes/055-sebastian-thiel.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Gitoxide with Sebastian Thiel"
+date: 2022-01-21T16:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e055-sebastian-thiel.mp3
+duration: "55:16"
+length: "54075439"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Sebastian Thiel](https://twitter.com/theelbasian), creator of [Gitoxide](https://github.com/Byron/gitoxide).
```

Comment:
Just a note for the future — I think we should start including a very brief summary of "the thing" as part of the intro text here. So something like:

> Allen Wyma talks with X, creator of Y, a [short description of Y].

Reply:
👍